packages/firebase_ml_vision/lib/src/firebase_vision.dart

```diff
@@ -14,6 +14,15 @@ enum ImageRotation { rotation0, rotation90, rotation180, rotation270 }
 /// Indicates whether a model is ran on device or in the cloud.
 enum ModelType { onDevice, cloud }
 
+/// Detected language from text recognition in regular and document images.
+class RecognizedLanguage {
```

Comment:
This class is moved to firebase_vision because it is a shared class between TextRecognizer and DocumentTextRecognizer and therefore should be in neither TextRecognizer's class file nor DocumentTextRecognizer's class file. FirebaseVisionImage which is also a class used by multiple detectors is residing here.

Alternatively we can create separate files for each such class or some sort of utils file.

Reply:
On the native side they FirebaseVisionImage is in `common`. However I think having them in `firebase_vision` is fine for now. It would be nice to have them mirror the native SDKs in functionality and in structure but this can be done in the future.